src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Waiting/VisualStudioWaitContextTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Shell.Interop;
+
+using Moq;
+
+using Xunit;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Waiting
+{
+    public class VisualStudioWaitContextTests
+    {
+        [Fact]
+        public static void SetPropertyAllowCancel_Test()
```

Comment:
AllowCancel_XXX, remove the test - we know its a test because it's marked with "[Fact]". :)